core/src/com/projectkorra/projectkorra/BendingPlayer.java

```diff
@@ -354,36 +339,23 @@ public static OfflineBendingPlayer getOrLoadOffline(@NotNull final OfflinePlayer
 		}
 	}
 
-	/**
-	 * Gets a BendingPlayer instance for the provided player
-	 * @param player The player
-	 * @return The BendingPlayer instance
-	 */
-	public static BendingPlayer getBendingPlayer(@NotNull final Player player) {
-		return getBendingPlayer((OfflinePlayer)player);
-	}
-
-
```

Comment:
Why is this removed?

Reply:
(In the PR desc)

Reply:
I actually would recommend not removing this function, as it is likely to break every single addon or plugin that calls this function directly.